src/views/Dashboard/Customer/Home/GeneticData/GeneticDataList.vue

```diff
@@ -14,7 +14,7 @@
 
       template(v-slot:[`item.uploadDate`]="{ item }")
         .d-flex.flex-column.genetic-data-list__upload
-          span
+          span {{ formatDate(item.createdAt)}}
```

Comment:
the method only works when the component is mounted/created. please update this one